src/pages/Invoice/Invoice.js

```diff
@@ -26,7 +28,7 @@ function Invoice() {
       });
   }, []);
 
-  if (!invoiceData?.id) return null;
+  if (!invoiceData?.id) return <LoadingPage />;
```

Comment:
invoice데이터가 없을때 로딩 컴포넌트를 보여주려는 의도인걸로 파악됩니다.
```suggestion
  if (!invoiceData) return <LoadingPage />;
```
로 수정할 수 있지않을까요?